scripts/use_node

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+
+# This script will find the appropriate Node.js runtime binary and execute it with any
+# parameters passed in.
+#
+# Set a variable named OSD_USE_NODE_JS_FILE_PATH to have it prefixed with OSD_HOME and executed
+# Example: SET OSD_USE_NODE_JS_FILE_PATH=\src\cli\dist
+#
+# NODE_OPTIONS is built using config/node.options and overridden by any previously set NODE_OPTIONS.
+# To pass in any specific defaults that can be overridden by both of them, use OSD_NODE_OPTS_PREFIX.
+
+SCRIPT="$0"
+
+OS=$(uname -s | awk '{print tolower}')
```

Comment:
`awk '{print tolower}'`

is failing for me so I don't get an OS. $OS outputs as empty for me on my Ubuntu

Reply:
Thanks. Peter made this suggestion. I will revert back to what we had before.

Reply:
I reverted this piece.

Reply:
@peterzhuamazon I had to revert this change.